### DISCUSSION TITLE
Vercel Blob access

### DISCUSSION THREAD

**Discussion by ab192130** [vercel/community#3599] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard/stores?type=blob

### Steps to Reproduce

Hi Vercel team and community members,

I've got an email about Vercel Blob access. Because I've registered to the waitlist earlier. 

But when I try to create a store in the specified store page, it throws: "The specified resource does not exist."

You can see my every step with pictures:

### Step 1:
![step 1](https://files.catbox.moe/fkkvv2.png)

### Step 2:
![step 2](https://files.catbox.moe/81yltl.png)

### Step 3:
**(Note: If you've noticed, It shows like I'm still in waitlist. But the email tells me opposite.. In addition, Continue button is enabled when I open the link directly from the email. It's disabled usually)**

![step 3](https://files.catbox.moe/qtjj8a.png)

### Step 4:
![step 4](https://files.catbox.moe/yu3jdn.png)

### Step 5:
![step 5](https://files.catbox.moe/siwr5i.png)

Here's the console logs:
![logs](https://files.catbox.moe/uzctb1.png)

I think it's a bug. Sorry, If I'm wrong.

Thanks for your attention

**Comment by v1s10n-4** [comment]
I'm in the exact same case, got the email, tried to create a blob store, storage type form is in a weird state ("continue" button clickable), same error when clicking "create"
